### PR TITLE
Test for var() evaluation in container size queries

### PR DESCRIPTION
**Pull request by mirisuzanne**
See https://github.com/w3c/csswg-drafts/issues/8088#issuecomment-1371590352 for the CSSWG resolution, and https://github.com/w3c/csswg-drafts/commit/ed383b552d3c977ad18116d8038c73d2c997f5a3 for the spec change. Variable references resolve based on the matched container.

### CODE REVIEW

File: css/css-contain/container-queries/var-evaluation.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<title>CSS Container Queries Test: var()</title>
+<link rel="help" href="https://drafts.csswg.org/css-contain-3/#size-container">
```
Comment (lilles) [inline css/css-contain/container-queries/var-evaluation.html:3]: Needs to be css-conditional-5 now. Also this file needs to move into the css-conditional/container-queries directory.

File: css/css-contain/container-queries/var-evaluation.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      height: 50px;
+      background-color: green;
+    }
+  }
+</style>
+<div id=container>
+  <div id=target></div>
+</div>
+<script>
+  setup(() => assert_implements_container_queries());
```
Comment (lilles) [inline css/css-contain/container-queries/var-evaluation.html:27]: This is now named 'assert_implements_size_container_queries()'

File: css/css-contain/container-queries/var-evaluation.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<title>CSS Container Queries Test: var()</title>
```
Comment (lilles) [inline css/css-contain/container-queries/var-evaluation.html:2]: How about  "var()" => "var() in size query values"?

**Review by lilles — COMMENTED** [review]
Sorry about the late review.

**Comment by mirisuzanne** [comment]
@lilles Thanks - I think I've addressed your comments here?

**Comment by dbaron** [comment]
Is it expected that the current test is [failing across engines](https://wpt.fyi/results/css/css-conditional/container-queries/var-evaluation.html?label=pr_head&max-count=1&pr=45135)?  (I think it seems likely to be expected.)

File: css/css-conditional/container-queries/var-evaluation.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<title>CSS Container Queries Test: var() in size query values</title>
+<link rel="help" href="https://drafts.csswg.org/css-conditional-5/#size-container">
+<link rel="help" href="https://drafts.csswg.org/mediaqueries-4/#units">
```
Comment (dbaron) [inline css/css-conditional/container-queries/var-evaluation.html:4]: Next time around, it probably doesn't hurt to also add a `<link rel="help" href="https://github.com/w3c/csswg-drafts/issues/8088">` pointing to the relevant github issue.  But I don't think we need to bother fixing it here.

**Comment by mirisuzanne** [comment]
@dbaron I would expect it, yes

File: css/css-conditional/container-queries/var-evaluation.html (hunk 4 above)
Comment (dbaron) [reply css/css-conditional/container-queries/var-evaluation.html:4]: Let's see if the UI lets me do it without using anything on the command line, though.
```suggestion
<link rel="help" href="https://drafts.csswg.org/mediaqueries-4/#units">
<link rel="help" href="https://github.com/w3c/csswg-drafts/issues/8088">
```

**Review by dbaron — APPROVED** [review]

**Pull request merged**